addons/l10n_cl/views/account_move_view.xml

```diff
@@ -49,7 +49,7 @@
         <field name="arch" type="xml">
             <tree decoration-info="state == 'draft'" default_order="create_date" string="Invoices and Refunds" decoration-muted="state == 'cancel'" js_class="account_tree">
                 <field name="l10n_latam_document_type_id_code"/>
-                <field name="l10n_latam_document_number" string="Folio"/>
+                <field name="name"/>
```

Comment:
I don't see the connection with this

Reply:
removed here: f9b4031da1670d22198fa17c4c6da88b4aef5adf